src/vivarium_public_health/risks/distributions.py

```diff
@@ -325,6 +398,35 @@ class DichotomousDistribution(RiskExposureDistribution):
     #################
 
     def build_all_lookup_tables(self, builder: "Builder") -> None:
+        """Build lookup tables for a binary risk exposure.
+
+        Lookup Tables Created
+        ---------------------
+        exposure
+            Probability of being in the exposed category (cat1). Values must
+            be in the range [0, 1].
+        paf
+            Population attributable fraction, initialized to 0.0. This can be
+            modified by risk effects to account for joint PAF calculations.
+
+        Data Sources Used
+        -----------------
+        - ``exposure``: Exposure probability for cat1. Can be:
+
+          - A scalar value (float) representing constant exposure probability
+          - A DataFrame with exposure probabilities by demographic group
+          - A DataFrame with polytomous categories that will be rebinned
+            (see ``rebinned_exposed`` configuration)
+
+        Notes
+        -----
+        If ``rebinned_exposed`` is configured, polytomous exposure data is
+        automatically rebinned: specified categories are combined into 'cat1'
+        (exposed) and remaining categories become 'cat2' (unexposed).
+
+        The PAF lookup table enables joint PAF calculations when multiple
+        risk effects modify the same target.
```

Comment:
This paf lookup table (and by extension the joint paf pipeline are here so that the exposure parameters pipeline can be the *target* of a risk effect. I think it'd be good to be clear about that here.